src/ferc_xbrl_extractor/datapackage.py

```diff
@@ -139,7 +139,7 @@ def __hash__(self):
 Map callables to schema field type to convert parsed values (Data Package `field.type`).
 """
 
-TABLE_NAME_PATTERN = re.compile("(.+) - Schedule - (.*)")  # noqa: W605, FS003
+TABLE_NAME_PATTERN = re.compile("(.+)\s+-\s+Schedule\s+-\s+(.*)", re.I)  # noqa: W605
```

Comment:
One of the tables has an extra space in the name, so I've switched from matching a single space exactly to just any whitespace, and made the pattern case-insensitive as another table had "schedule" spelled all lowercase.